app/lib/frontend/templates.dart

```diff
@@ -543,9 +543,20 @@ class TemplateService {
     String platform,
   ) {
     final platformDict = getPlatformDict(platform);
+    final packagesUrl = platform == null ? '/packages' : '/$platform/packages';
+    final morePackages = 'More ${platformDict.name} packages...';
+    final moreLinks = <String>[
+      '<a href="$packagesUrl">${_htmlEscaper.convert(morePackages)}</a>'
+    ];
+    if (platform == KnownPlatforms.flutter) {
+      final url = new Uri(
+          path: '/packages', queryParameters: {'q': 'dependency:flutter'});
+      final flutterOnly = 'Flutter-specific packages...';
+      moreLinks.add(
+          '<a href="${_attrEscaper.convert(url.toString())}">${_htmlEscaper.convert(flutterOnly)}</a>');
```

Comment:
Could inline $flutterOnly here. There's not much reason to escape it, since it's our own constant.

We could technically make the entire string a constant, since the url is also hard-coded.